app/main/posts/savedsearches/savedsearches-controller.js

```diff
@@ -43,7 +43,10 @@ module.exports = [
 
         // Set initial filter state
         PostFilters.setMode('savedsearch', savedSearch.id);
-        PostFilters.setFilters(savedSearch.filter);
+
+        var filters = savedSearch.filter;
+        filters.saved_search = savedSearch.id;
```

Comment:
I think this can be removed. It's not doing anything there because we don't use the saved search in the filter query, and might be breaking something since it should be an object, unless we can just go back to how it was (with IDs) which would be best i think..